charts/seed-monitoring/charts/prometheus/rules-tests/kube-etcd3.rules.test.yaml

```diff
@@ -20,7 +20,7 @@ tests:
     values: '0+0x30'
   # KubeEtcd3HighNumberOfFailedProposals
   - series: 'etcd_server_proposals_failed_total{job="kube-etcd3", pod="etcd"}'
-    values: '0+0x15 1+0x15 2+0x15 3+0x15 4+0x15 5+0x15 6+0x15 7+0x15'
+    values: '0+1x85'
```

Comment:
`values: '0+1x85'` spans values over 85*30=42.5min. Becuase of which in description the values recived is 85.5. The additional 0.5 is due to unavailability of values in rest of 17.5min. This makes it hard to understand the test scenario. 
So, Small correction here, will you please make it : 
`values: '0+1x81 0+0x39'` 
This spans values over entire 1 hour ((81+39)*30sec) period. So, expected failed proposal count could be precisely calculated as 81.

Reply:
Done